plugin/src/main/groovy/com/github/kright/androidscalasupport/MultidexExtension.groovy

```diff
@@ -0,0 +1,61 @@
+package com.github.kright.androidscalasupport
+
+import org.gradle.api.Project
+
+/**
+ * Created by lgor on 01.03.2017.
+ */
+class MultidexExtension {
+
+	private final Project project
+
+	private MainDexOverwrite mainDexModifier = null
```

Comment:
usually you want the class and the method named the same if theres no particular reason not to